### DISCUSSION TITLE
Interesting!

### DISCUSSION THREAD

**Discussion by breck7** [listiary/DescribeCompiler#3] [General]
Love it when I come across the rare fellow ANTLR fan!

Sad I couldn't give it a try. Would love to see it in Javascript with a web repl!

Here's my user test (of the github readme :)):
https://news.pub/index.html?try=https://www.youtube.com/embed/-L6V9Bvdvvg?si=FVIUgJZBgj8HRumV

**Comment by viktorchernev** [comment]
The project can be deployed as a microservice on Amazon Web Services (AWS) using Lambda. In this setup, it’s accessible via HTTP POST requests from JavaScript.

Setting it up requires registering for a free-tier AWS account, installing Visual Studio, and deploying the code. However, I have it hosted on my own account for convenience. If you’d like to test it from JavaScript, feel free to reach out, and I’ll share a URL. You can contact me here or via email at [vchernev91@abv.bg](mailto:vchernev91@abv.bg).

Additionally, I’m working on an online JavaScript-based editor / compiler for the language, which I plan to release in the future. Stay tuned for updates!

Best regards from Bulgaria,
Viktor Chernev
